Gemfile

```diff
@@ -17,9 +17,6 @@ gem "puma", ">= 5.0"
 # Use Active Model has_secure_password [https://guides.rubyonrails.org/active_model_basics.html#securepassword]
 # gem "bcrypt", "~> 3.1.7"
 
-# Windows does not include zoneinfo files, so bundle the tzinfo-data gem
-gem "tzinfo-data", platforms: %i[ windows jruby ]
```

Comment:
reason?

Reply:
We won't be needing it in docker, also windows isn't valid platform according to rvm. I think they use mingw